docs/README.md

```
@@ -0,0 +1,38 @@
+
+1. [معرفی و مفاهیم کلیدی](01-introduction.md)
+2. [نصب و راه‌اندازی](02-installation.md)
+3. [ویژگی‌های اصلی](03-features.md)
+4. [میانبرها و تنظیمات](04-shortcuts-settings.md)
+5. [قوانین و حافظه‌ها](05-rules-memories.md)
+6. [ادغام با ابزارها](06-integrations.md)
+7. [نکات پیشرفته](07-advanced-tips.md)
+
+## شروع سریع
+
+### 1. دانلود و نصب
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)